openreview/conference/helpers.py

```diff
@@ -31,6 +31,9 @@ def get_venue(client, venue_id, support_user='OpenReview.net/Support'):
     venue.allow_gurobi_solver = domain.content.get('allow_gurobi_solver', {}).get('value', False)
     venue.preferred_emails_groups = domain.content.get('preferred_emails_groups', [])
     
+    venue.submission_stage = openreview.stages.SubmissionStage(
```

Comment:
@melisabok Using the venue code to upload decisions works, but I had to add the submission_stage here. `submission_stage.name` is used when we call `get_paper_group_prefix()` to get the paper decision invitation. Is there a better way to do this?